kubetest2/internal/deployers/eksapi/deployer.go

```diff
@@ -60,6 +61,7 @@ type deployer struct {
 }
 
 type deployerOptions struct {
+	Addons                      []string      `flag:"addons" desc:"Managed addons (name:version pairs) to create in the cluster. Use 'latest' for the most recent version, or 'default' for the default version."`
```

Comment:
I wanted to use a `map[string]string` here to get the parsing of the pairs for free, but it didn't work. 🤷